components/EmbraceModal.vue

```diff
@@ -1,16 +1,14 @@
 <template>
   <ModalWithBackdrop v-model="showModal">
     <form v-if="!completionMarked" class="w-full h-full overflow-y-auto p-4 md:p-6 mb-2 md:mb-6 mt-6 text-slate-800 text-left">
-      <div class="flex flex-row items-center">
+      <button class="flex flex-row items-center cursor-pointer hover:opacity-80" @click="closeModal">
```

Comment:
When the "back to form" is pressed, the main page is refreshed and all entries are removed. However, you can still have all your entries when you press the 'Cancel' button. Shouldn't it be the opposite?

Reply:
@PooriaT - if this is still an issue could you please open a ticket?